src/edgepi/gpio/edgepi_gpio_chip.py

```diff
@@ -39,11 +39,10 @@ def read_gpio_pin_state(self, pin_name: str = None):
         Returns:
             `bool`: True if state is high, False if state is low
         """
-        self.open_gpio(pin_num=self.__pin_name_dict[pin_name],
+        with self.open_gpio(pin_num=self.__pin_name_dict[pin_name],
                        pin_dir=self.gpiochip_pins_dict[pin_name].dir,
-                       pin_bias=self.gpiochip_pins_dict[pin_name].bias)
-        state = self.read_state()
-        self.close_gpio()
+                       pin_bias=self.gpiochip_pins_dict[pin_name].bias):
+            state = self.read_state()
```

Comment:
what does `pin_bias` mean?

Reply:
pin_bias means the pull direction, either pull-down or pull-up